eng/bomgenerator/validateAzureSpringBootBom.cmd

```diff
@@ -1,3 +0,0 @@
-set "bomPomFileLocation=%~dp0..\..\sdk\boms\azure-spring-boot-bom\pom.xml"
-set "reportFileLocation=%~dp0report.html"
-mvn clean install && mvn exec:java -Dexec.args="-mode=analyze -pomFile=%bomPomFileLocation% -reportFile=%reportFileLocation%" && copy %~dp0report.html azurespringbootbomreport.html
```

Comment:
Please update `eng/bomgenerator/validateAzureSpringCloudBom.cmd`. Including the filename and `bomPomFileLocation`.

Reply:
There's already https://github.com/Azure/azure-sdk-for-java/blob/main/eng/bomgenerator/validateAzureSpringCloudBom.cmd, just removed the unused one, but seems this one is not referenced by any file.

Reply:
> There's already https://github.com/Azure/azure-sdk-for-java/blob/main/eng/bomgenerator/validateAzureSpringCloudBom.cmd,

I mean do like this: 
![image](https://user-images.githubusercontent.com/13167207/186806043-c32fa141-7159-4878-9024-89d14ed17a6c.png)

>  just removed the unused one, but seems this one is not referenced by any file.

It's a script file, maybe it should be executed manually.

Reply:
Hi @pallavit, please note this file name and pom file path should be updated.